src/modules/Elsa.Workflows.Core/Contracts/IIdentityGraphService.cs

```diff
@@ -21,17 +21,17 @@ public interface IIdentityGraphService
     /// <summary>
     /// Assign unique identities tot the specified <see cref="ActivityNode"/> graph.
     /// </summary>
-    void AssignIdentities(ActivityNode root);
+    Task AssignIdentities(ActivityNode root);
```

Comment:
Please mind the naming convention to apply the `Async` suffix when changing methods into async.